### PR TITLE
chore: update benchmarks

### PR DESCRIPTION
**Pull request by achingbrain**
The bugs stopping WebRTC and WebTransport from being testable in browsers have been fixed and shipped so they can be added to the suite.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: benchmarks/transports/src/tests.ts [hunk 1]
```diff
@@ -99,6 +96,10 @@ const tcpImpls: Record<string, Impl> = {
 function addTests (name: string, impls: Record<string, Impl>, tests: Test[], relay: Multiaddr): void {
   for (const [implAName, implA] of Object.entries(impls)) {
     for (const [implBName, implB] of Object.entries(impls)) {
+      if (implAName !== implBName) {
+        continue
+      }
```
Comment (SgtPooki) [inline benchmarks/transports/src/tests.ts:101]: this block will ensure we're only testing an implementation against itself. Is this supposed to be `===`?

**Review by SgtPooki — CHANGES_REQUESTED** [review]

File: benchmarks/transports/src/tests.ts (hunk 1 above)
Comment (achingbrain) [reply benchmarks/transports/src/tests.ts:101]: Ah, no - left that in by accident

File: benchmarks/transports/src/tests.ts [hunk 2]
```diff
@@ -99,6 +96,10 @@ const tcpImpls: Record<string, Impl> = {
...
+
```
Comment (achingbrain) [inline benchmarks/transports/src/tests.ts:102]:
```suggestion
```

File: benchmarks/transports/src/runner/helia/recipient.ts [hunk 3]
```diff
@@ -27,7 +27,8 @@ try {
   }))
 
   console.info(`TEST-OUTPUT:${Date.now() - start}`)
-} catch {
+} catch (err: any) {
+  console.error(err)
```
Comment (achingbrain) [inline benchmarks/transports/src/runner/helia/recipient.ts:31]:
```suggestion
} catch {
```

**Pull request merged**